source/transfer_list.rst

```diff
@@ -902,5 +903,128 @@ This manifest contains the SPMC attribute node consumed by the SPMD
      - hdr_size
      - Holds a SPMC manifest image in DT format.
 
+.. _dice_entries:
+
+Entries related to DICE
+^^^^^^^^^^^^^^^^^^^^^^^
+
+These entries hold [DICE]_ and [Open-DICE]_ information elements that needs to be transferred from one boot stage to the next.
+
+This includes the CDIs used for attestation (and optional sealing), as well as the accumulated certificate chain.
+
+The following entry types are currently defined:
+
+**CDI for attestation (XFERLIST_DICE_CDI_ATTEST)**
+
+This entry type holds the CDI described in [DICE]_.
+In [Open-DICE]_ it is referred to as "Attestation CDI".
+
+The CDI computed by the Sender's boot stage SHALL be added to the TL.
+The CDI received from the previous boot stage (i.e., DICE layer) MUST NOT be forwarded.
+
+.. _tab_dice_cdi_attest:
+.. list-table:: DICE attestation CDI
```

Comment:
So, I am agree with adding the openDICE profile. Would it make sense to correlate this one to the TCG Alias ID?
The attestation CDI seems similar to the Alias one described in the spec.

Reply:
> So, I agree with adding the openDICE profile. Would it make sense to correlate this one to the TCG Alias ID? The attestation CDI seems similar to the Alias one described in the spec.

The Alias Key described in [Section 8 of DICE LA](https://trustedcomputinggroup.org/wp-content/uploads/DICE-Layering-Architecture-r19_pub.pdf#page=18) is the leaf key in the DICE cert chain, i.e., the one that is obtained from the last CDI in the TCB's components "stack".  This key is computed (also) using the TCI (i.e., at a minimum, the digest of the firmware code) of the last layer.  So it's not as stable as Open-DICE [sealing key](https://pigweed.googlesource.com/open-dice/+/0b5044098bf9b40128927d675dea4ec1fb75c510/docs/specification.md#cdi-values), which is computed from authority data, mode decision, and hidden inputs, i.e., all stable inputs.

So, structurally TCG Alias Key and Open-DICE sealing keys are quite different objects.  Also semantically: an Alias Key is the one that signs attestation challenges, while a sealing key is a key that encrypts some data (and binds that encryption to a specific state of the device/program).

Reply:
I wasn't referring to the sealing key, but it wasn't entirely clear. 
I meant the alias ID and the attestation one described in openDICE seems similar

Reply:
> I wasn't referring to the sealing key, but it wasn't entirely clear

ah, sorry I misunderstood.

> I meant the alias ID and the attestation one described in openDICE seem similar

In TCG, the alias is the leaf cert, so I guess by "alias ID" you mean the CDI associated with the last TCB layer?  If so, that's a very specific CDI.  Instead, the "attestation CDI" in Open-DICE is a generic name for *any* CDI computed by DICE.  It's the equivalent of TCG's CDI - without qualifications.

Reply:
The 'alias ID' parlance comes from https://www.microsoft.com/en-us/research/project/dice-device-identifier-composition-engine/. Looking at it, it seems to include measurements of various layers not only the last one. IOW it seems to measure all subsequent layers

Reply:
> The 'alias ID' parlance comes from https://www.microsoft.com/en-us/research/project/dice-device-identifier-composition-engine/. Looking at it, it seems to include measurements of various layers not only the last one. IOW it seems to measure all subsequent layers

OK, we were looking at different specs :-)

Yes, in DICE/RiOT sense you are right.  But a few years ago (about 8 I think) MSFT gave TCG change control over DICE.  And TCG has made some adjustments to the vocabulary, including narrowing the concept of "alias" to the outer layer only.

It is very confusing that MSFT hasn't clearly marked the RiOT spec as stale :-(